skimage/restoration/_rolling_ball.py

```diff
@@ -5,29 +5,30 @@
 
 
 def rolling_ball(image, *, radius=100, kernel=None, nansafe=False, num_threads=None):
-    """Estimate background intensity by rolling/translating a kernel.
+    """Estimate background intensity by translating a kernel over an image.
 
-    This rolling ball algorithm estimates background intensity for a
-    ndimage in case of uneven exposure. It is a generalization of the
-    frequently used rolling ball algorithm [1]_.
+    This function estimates the background intensity of an n-dimensional
+    image. Typically, it is useful for background subtraction in case of
+    uneven exposure. It is a generalization of the well-known rolling-ball
+    algorithm [1]_.
```

Comment:
```suggestion
    This function is a generalization of the rolling-ball algorithm [1]_
    to estimate the background intensity of an N-dimensional
    image. This is useful for background subtraction in case of
    uneven exposure. Think of the image as a landscape
    (where altitude is determined by intensity), under which
    a ball of given radius is rolled. At each position, the ball's
    apex determines the resulting background intensity.
```

Reply:
@stefanv I included this in 13eeb197b9381c60ae6d8326d8037962337d0c34. I wrote "n-dimensional" instead of "N-dimensional" although we use both conventions throughout the codebase; in this case, I think it's better to differentiate it from the "N" used in the Notes when referring to the algorithm's polynomial complexity.

Reply:
No problem; we should strive to get it uniform eventually, either n/N fine.